commit 1e02e3f790309e478e8ddd7ae0afeef0ec74e61b (refs/changes/11/5511/1)
Author: Tim Foster <tim.foster@joyent.com>
Date:   2019-02-07T19:47:03+00:00 (8 months ago)
    
    TRITON-1177 hermes build looking for node in all the wrong places (fix build)

diff --git a/Makefile b/Makefile
index 08000b3..bd361e9 100644
--- a/Makefile
+++ b/Makefile
@@ -166,8 +166,13 @@ $(DESTDIR)$(PREFIX)/sapi_manifests/%: $(PWD)/sapi_manifests/%
 	@mkdir -p `dirname $@`
 	cp $^ $@
 
+$(DESTDIR)$(PREFIX)/node_modules: $(STAMP_NODE_MODULES)
+	rm -rf $@
+	cp -r $(PWD)/node_modules $@
+
 clean::
 	rm -rf $(PWD)/node_modules
+	rm -rf $(PWD)/make_stamps/node-modules
 	rm -rf $(PWD)/proto
 
 include ./tools/mk/Makefile.deps
